content/automate/ManageIQ/Infrastructure/VM/Transform/Import.class/__methods__/list_tag_names.rb

```diff
@@ -19,7 +19,7 @@ def main
                   category_name = @handle.root['dialog_tag_category']
                   if category_name.present?
                     @handle.log(:info, "Selected tag category: #{category_name}")
-                    category = @handle.vmdb(:classification).find_by_name(category_name)
+                    category = @handle.vmdb(:classification).lookup_by_name(category_name)
```

Comment:
@mkanoor @tinaafitz The `find_by_*` calls work today because the automate engine allows these specific calls to pass through, otherwise they need to be specified with `expose` or similar technique in the service_models.

See https://github.com/lfu/manageiq-automation_engine/blob/master/lib/miq_automation_engine/engine/miq_ae_method_service/miq_ae_service_model_base.rb#L27-L30

Should it now include `lookup_by` and the depreciation side will require some thinking.  Not sure it should hold up these PRs.

Reply:
@gmcculloug This is ok since its our automate methods, we just have to aware of customers scripts and if they use the find_by_* methods.